library/src/symbols/sym.rs

```diff
@@ -218,10 +218,23 @@ pub(crate) const SYM: &[(&str, Symbol)] = symbols! {
         quest: '≟',
         small: '﹦',
         succ: '⋟',
+        triple: '≡',
+        quad: '≣',
     ],
     gt: [
         '>',
         circle: '⧁',
+        curly: '≻',
+        curly.approx: '⪸',
+        curly.double: '⪼',
+        curly.eq: '≽',
+        curly.eq.not: '⋡',
+        curly.eqq: '⪴',
+        curly.napprox: '⪺',
+        curly.neqq: '⪶',
```

Comment:
We should also rename neqq, eqq, etc to nequiv, equiv, etc, to match where relevant, in my opinion.